### PR TITLE
run rustfmt on libcollections test module

### PR DESCRIPTION
**Pull request by srinivasreddy**

### CODE REVIEW

**Comment by srinivasreddy** [comment]
cc @nrc

File: src/libcollectionstest/enum_set.rs [hunk 1]
```diff
@@ -60,9 +62,8 @@ fn test_len() {
     assert_eq!(e.len(), 0);
 }
 
-///////////////////////////////////////////////////////////////////////////
-// intersect
-
+/// ////////////////////////////////////////////////////////////////////////
```
Comment (nrc) [inline src/libcollectionstest/enum_set.rs]: Could you manually fixup these comment changes please (to undo what rustfmt has done)

File: src/libcollectionstest/enum_set.rs [hunk 2]
```diff
@@ -221,13 +219,76 @@ fn test_overflow() {
...
     #[repr(usize)]
     enum Bar {
-        V00, V01, V02, V03, V04, V05, V06, V07, V08, V09,
-        V10, V11, V12, V13, V14, V15, V16, V17, V18, V19,
-        V20, V21, V22, V23, V24, V25, V26, V27, V28, V29,
-        V30, V31, V32, V33, V34, V35, V36, V37, V38, V39,
-        V40, V41, V42, V43, V44, V45, V46, V47, V48, V49,
-        V50, V51, V52, V53, V54, V55, V56, V57, V58, V59,
-        V60, V61, V62, V63, V64, V65, V66, V67, V68, V69,
+        V00,
```
Comment (nrc) [inline src/libcollectionstest/enum_set.rs]: this should be undone, and add a rustfmt_skip

File: src/libcollectionstest/slice.rs [hunk 3]
```diff
@@ -336,10 +340,9 @@ fn test_dedup_unique() {
     v1.dedup();
     let mut v2: Vec<Box<_>> = vec![box 1, box 2, box 3, box 3];
     v2.dedup();
-    /*
-     * If the boxed pointers were leaked or otherwise misused, valgrind
-     * and/or rt should raise errors.
-     */
+    // If the boxed pointers were leaked or otherwise misused, valgrind
+    // and/or rt should raise errors.
+    //
```
Comment (nrc) [inline src/libcollectionstest/slice.rs]: Could you remove the last, empty line please

File: src/libcollectionstest/slice.rs [hunk 4]
```diff
@@ -350,10 +353,9 @@ fn test_dedup_shared() {
     v1.dedup();
     let mut v2: Vec<Box<_>> = vec![box 1, box 2, box 3, box 3];
     v2.dedup();
-    /*
-     * If the pointers were leaked or otherwise misused, valgrind and/or
-     * rt should raise errors.
-     */
+    // If the pointers were leaked or otherwise misused, valgrind and/or
+    // rt should raise errors.
+    //
```
Comment (nrc) [inline src/libcollectionstest/slice.rs]: here too

**Comment by nrc** [comment]
r+ with the inline comments addressed

**Comment by srinivasreddy** [comment]
@nrc done. Thanks

**Comment by nrc** [comment]
Thanks!

@bors: r+

File: src/libcollectionstest/enum_set.rs [hunk 5]
```diff
@@ -220,6 +222,7 @@ fn test_overflow() {
     #[allow(dead_code)]
     #[derive(Copy, Clone)]
     #[repr(usize)]
+    #[rustfmt_skip]
```
Comment (alex-ozdemir) [inline src/libcollectionstest/enum_set.rs]: Seems like this caused the build problem.
The error can be seen [here](https://travis-ci.org/rust-lang/rust/builds/132171217#L3552).

**Comment by srinivasreddy** [comment]
@nrc  removed #[rustfmt_skip] . Please approve. Thanks

**Pull request merged**